frontends/mit-open/webpack.config.js

```diff
@@ -151,8 +178,9 @@ module.exports = (env, argv) => {
         },
       }),
       new webpack.EnvironmentPlugin({
-        ENVIRONMENT: "production",
-        MITOPEN_AXIOS_BASE_PATH: undefined,
+        // within app, define process.env.VAR_NAME with default from cleanEnv
+        MITOPEN_AXIOS_BASE_PATH,
+        ENVIRONMENT,
```

Comment:
The old syntax was very, very confusing:
- ` MITOPEN_AXIOS_BASE_PATH: undefined,` meant that webpack would error if `MITOPEN_AXIOS_BASE_PATH` was undefined
- `ENVIRONMENT: "production"` meant "use the environment variable, but default to `'production'`.

With envalid, the same thing happens via default or devDefault, but it's much clearer IMO.